source/cipher_openssl.c

```diff
@@ -727,10 +727,8 @@ int aws_cryptosdk_sig_verify_start(
     struct aws_allocator *alloc,
     const struct aws_string *pub_key,
     const struct aws_cryptosdk_alg_properties *props) {
-    AWS_PRECONDITION(pctx);
-    AWS_PRECONDITION(alloc);
     AWS_PRECONDITION(aws_string_is_valid(pub_key));
-    AWS_PRECONDITION(props);
+    AWS_PRECONDITION(props != NULL);
```

Comment:
Why not `aws_cryptosdk_alg_properties_is_valid()`?

Reply:
Updated to this!